### PR TITLE
added instructions on best way to get graphviz with python bindings on os X

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: easybuild/framework/easyconfig/tools.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
     sys.path.append('/usr/lib64/graphviz/python/')
     import gv
 except ImportError, err:
-    graph_errors.append("Failed to import graphviz: try yum install graphviz-python, or apt-get install python-pygraphviz")
+    graph_errors.append("Failed to import graphviz: try yum install graphviz-python, or apt-get install python-pygraphviz, or brew install graphviz --with-bindings")
```
Comment (boegel) [inline easybuild/framework/easyconfig/tools.py]: fix long line...

**Comment by boegel** [comment]
Thanks @JensTimmerman!

**Comment by JensTimmerman** [comment]
maybe this should show something like: eb graphviz-xxx.eb :p

**Comment by boegel** [comment]
patches welcome ;)

**Pull request merged**